README.md

```diff
@@ -54,33 +54,44 @@ installation/runtime requirements.
-                               on macOS and Linux
-  -s, --sequential             Rebuild modules sequentially, this is enabled by
-                               default on Windows
-  -o, --only                   Only build specified module, or comma separated
-                               list of modules. All others are ignored.
-  -b, --debug                  Build debug version of modules
-  --prebuild-tag-prefix        GitHub tag prefix passed to prebuild-install.
-                               Default is "v"
-
-Copyright 2016
+  -v, --version                  The version of Electron to build against
+                                                                        [string]
+  -f, --force                    Force rebuilding modules, even if we would skip
+                                 it otherwise                          [boolean]
+  -a, --arch                     Override the target architecture to something
+                                 other than your system's               [string]
+  -m, --module-dir               The path to the node_modules directory to
+                                 rebuild                                [string]
+  -w, --which-module             A specific module to build, or comma separated
+                                 list of modules. Modules will only be rebuilt
+                                 if they also match the types of dependencies
+                                 being rebuilt (see --types).           [string]
+  -o, --only                     Only build specified module, or comma separated
+                                 list of modules. All others are ignored.
+                                                                        [string]
+  -e, --electron-prebuilt-dir    The path to the prebuilt electron module
+                                                                        [string]
+  -d, --dist-url                 Custom header tarball URL              [string]
+  -t, --types                    The types of dependencies to rebuild.  Comma
+                                 separated list of "prod", "dev" and "optional".
+                                 Default is "prod,optional"             [string]
+  -p, --parallel                 Rebuild in parallel, this is enabled by default
+                                 on macOS and Linux                    [boolean]
+  -s, --sequential               Rebuild modules sequentially, this is enabled
+                                 by default on Windows                 [boolean]
+  -b, --debug                    Build debug version of modules        [boolean]
+      --prebuild-tag-prefix      GitHub tag prefix passed to prebuild-install.
+                                 Default is "v"                         [string]
+      --force-build-from-source  Skip prebuild download and rebuild module from
+                                 source. Default is false              [boolean]
```

Comment:
issue: I think we should match the argument name that we have in `RebuildOptions`, the CLI expects an argument called `buildFromSource` so passing `--force-build-from-source=true` has no effect right now: https://github.com/electron/rebuild/blob/a93581c57d7bf9f1408693fc40614ccc10f51488/src/cli.ts#L131

Reply:
🤔 Just regenerated those based on the output from `node lib/cli.js --help`

Reply:
ah `buildFromSource` is not listed as an option here, so it doesn't show up in the help output:

https://github.com/electron/rebuild/blob/b8ac6dbb3eed2a133b41700ab36940c28e9264e3/src/cli.ts#L14-L34

Since we do list `buildFromSource` in `RebuildOptions`, I think we just forgot to add it to the CLI options when it was first implemented?

Reply:
Good point: [ae9de35](https://github.com/electron/rebuild/pull/1141/commits/ae9de351f9f413ec1b60ea3ef82b852ead63dddb)